src/applications/personalization/dashboard/tests/helpers.js

```diff
@@ -2,3 +2,857 @@ import environment from '~/platform/utilities/environment';
 
 export const shouldMockApiRequest = () =>
   environment.isLocalhost() && !window.Cypress && !window.VetsGov.pollTimeout;
+
+export const mockAppealData = {
```

Comment:
This big ole mock data object might fit better as a json file in the fixtures folder, since it is only ever used within the getsStatusContents unit test.